sdk/core/core-http/review/core-http.api.md

```diff
@@ -74,59 +74,65 @@ export interface BaseMapper {
     xmlNamespacePrefix?: string;
 }
 
-// @public (undocumented)
+// @public
 export abstract class BaseRequestPolicy implements RequestPolicy {
-    protected constructor(_nextPolicy: RequestPolicy, _options: RequestPolicyOptionsLike);
+    protected constructor(
+    _nextPolicy: RequestPolicy,
+    _options: RequestPolicyOptionsLike);
     log(logLevel: HttpPipelineLogLevel, message: string): void;
-    // (undocumented)
     readonly _nextPolicy: RequestPolicy;
-    // (undocumented)
     readonly _options: RequestPolicyOptionsLike;
-    // (undocumented)
     abstract sendRequest(webResource: WebResourceLike): Promise<HttpOperationResponse>;
     shouldLog(logLevel: HttpPipelineLogLevel): boolean;
 }
 
-// @public (undocumented)
+// @public
 export class BasicAuthenticationCredentials implements ServiceClientCredentials {
     constructor(userName: string, password: string, authorizationScheme?: string);
-    // (undocumented)
     authorizationScheme: string;
-    // (undocumented)
     password: string;
     signRequest(webResource: WebResourceLike): Promise<WebResourceLike>;
-    // (undocumented)
     userName: string;
 }
 
 // @public
 export function bearerTokenAuthenticationPolicy(credential: TokenCredential, scopes: string | string[]): RequestPolicyFactory;
 
-// @public (undocumented)
+// @public
+export type CommonRequestInfo = string;
```

Comment:
Necessary because of previous warning:

```
    // Warning: (ae-forgotten-export) The symbol "CommonRequestInfo" needs to be exported by the entry point coreHttp.d.ts
```